develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md

```diff
@@ -194,22 +194,48 @@ For more information, refer to [Multi-threaded timers SNMP](xref:AdvancedMultiTh
 
 ### ssh pwd
 
-This option specifies that this parameter holds the password used for setting up SSH communication based on user credentials. When using this option, there should also be another parameter that uses the "ssh username" option. The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
+> [!IMPORTANT]
+> Link this parameter via the [Password](xref:Protocol.PortSettings.SSH.Credentials.Password) element 
```

Comment:
I would rewrite this as 
The usage of this option is not recommended. It only support connectors with a single SSH connection using the fixed port 22.  Instead, link this parameter via the [Password](xref:Protocol.PortSettings.SSH.Credentials.Password) element.

Reply:
Applied feedback